src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.10.0-preview",
+  "version": "1.0.0",
```

Comment:
We still require the package to be in preview when breaking changes are introduced to the SDK, and after enough time to ensure it is stable, you can remove the `-preview` suffix and re-publish a new version